Source/Orts.Simulation/Common/Commands.cs

```diff
@@ -1573,6 +1573,30 @@ public override void Redo()
 
 
     // Steam controls
+
+    // Steam booster command
+    [Serializable()]
+    public sealed class ContinuousSteamBoosterCommand : ContinuousCommand
+    {
+        public static MSTSSteamLocomotive Receiver { get; set; }
+
+        public ContinuousSteamBoosterCommand(CommandLog log, int injector, bool toState, float? target, double startTime)
+            : base(log, toState, target, startTime)
+        {
+            Redo();
+        }
+
+        public override void Redo()
+        {
+            if (Receiver == null) return;
+            {
+                Receiver.SteamBoosterChangeTo(ToState, Target);
+            }
```

Comment:
```suggestion
            Receiver?.SteamBoosterChangeTo(ToState, Target);
```

Reply:
I'm sorry but I won't approve it as is. It looks confusing to me. In particular, at first glance your code seems to do
```csharp
if (Receiver == null)
{
    Receiver.SteamBoosterChangeTo(ToState, Target);
}
```
until you notice the "return" statement. The curly braces are completely misleading.

The ?. operator is my preferred choice in this case, and is also recommended by ORMT (see http://www.elvastower.com/forums/index.php?/topic/35509-cs-null-conditional-operator/). It is also used elsewhere in Commands.cs, so your argument about consistency is not really valid. In my opinion, new code should use it whenever possible.

However, my intention is not to force you to use that syntax. It was just my suggestion. I have no problem if you prefer to use "standard" syntax, as long as the misleading braces are removed. Other acceptable solutions for me are:
```csharp
if (Receiver == null) return;
Receiver.SteamBoosterChangeTo(ToState, Target);
```
or
```csharp
if (Receiver != null)
{
    Receiver.SteamBoosterChangeTo(ToState, Target);
}
```
or
```csharp
if (Receiver != null) Receiver.SteamBoosterChangeTo(ToState, Target);
```

Reply:
Ok, I have adopted one of your suggested solutions.

Reply:
I'm sorry for being so nitpicking, but the indentation of `Receiver.SteamBoosterChangeTo(ToState, Target);` is still incorrect. I think it is important that we follow the coding guidelines to make code more readable for others. I could have corrected it myself, but you disabled "allow edits from maintainers".

Reply:
I have moved it across now. Thanks for checking.